packages/css/src/components/document/document.scss

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
```

Comment:
I found this in the create-vite-app scaffold, should we add it?

```css
:root {
  font-synthesis: none;
  text-rendering: optimizeLegibility;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  -webkit-text-size-adjust: 100%;
}
```

Reply:
- `font-synthesis: none`: Is our font still missing subscript and superscript? And small caps is probably missing as well. I don't really see the need to disable font-synthesis. 
- `text-rendering: optimizeLegibility`: I think Chrome uses this by default, but I think in general it makes sense for us to optimize legibility over speed.
- `-webkit-font-smoothing: antialiased` and `-moz-osx-font-smoothing: grayscale`: This is somewhat of a contested topic I think. I'd let the browser decide for now, and add it if that really causes issues. 
- `-webkit-text-size-adjust`: We add this on a component basis, but I think it's useful to add this on the document level as well, for text that doesn't use our DS components.

Reply:
Alright, so we want `text-rendering: optimizeLegibility` and `-webkit-text-size-adjust: 100%;`. On the latter, [this article](https://kilianvalkhof.com/2022/css-html/your-css-reset-needs-text-size-adjust-probably/) suggests adding two other prefix variations and using a value of `none` instead of `100%`.

We can follow one of three approaches:
1. Add both to `:root` or `.ams-html` or `.ams-body` or `.ams-screen` and use inheritance.
2. Declare for each component separately as we do now.
3. Same, but through a common mixin.

I would go for the latter; it is more in line with our component architecture but reuses the actual declarations as we do for other concepts already. Agree?

Reply:
Yeah, three is definitely a good idea. We could also add it to `:root` or `.ams-html` or `.ams-body` or `.ams-screen`, so components outside of the Design System still benefit from it? But that might be overkill